test/integration/targets/noexec/runme.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eux
+
+trap 'umount "${OUTPUT_DIR}/ramdisk"' EXIT
```

Comment:
I think it's more predictable to render the variable on the declaration stage.
```suggestion
trap "umount '${OUTPUT_DIR}/ramdisk'" EXIT
```